### DISCUSSION TITLE
Development, Test, Production

### DISCUSSION THREAD

**Discussion by rokahr** [Azure/data-management-zone#133] [Ideas] [answered]
After having done several deployments I think it would be beneficial if you could briefly describe what is going to change between Dev, Test and Prod deployment. Do we deploy different resources, is it only the naming, do we have changes in policies etc.? It's just an idea and more towards the UI

**Comment by marvinbuss** [comment]
Thanks @rokahr! Currently, this has no effect the setup and is purely used for the naming convention. I would add a note once that changes. Should we call this out?
Does this clarify the effect or do we need additional explanations?

![image](https://user-images.githubusercontent.com/34542414/127654557-2980aeb5-a00e-44a5-bf3f-a419b0b1986f.png)

**Reply by rokahr** [reply]
Specify a prefix and select an environment (Development, Test, Production) which will both be used as a prefix for all resource names. Independent of the environment, the same resources get deployed.

How does this sound?

**Reply by marvinbuss** [reply]
Thanks. Let me include this.

**Reply by marvinbuss** [reply]
This will be released when this PR gets merged: https://github.com/Azure/data-management-zone/pull/132

**Comment by marvinbuss** [comment, marked as answer]
I created an Issue for this discussion, where you can track when this gets merged: https://github.com/Azure/data-management-zone/issues/134
